apps/participants/forms.py

```diff
@@ -79,3 +82,56 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         if team:
             self.fields["experiment"].queryset = Experiment.objects.filter(team=team, working_version__isnull=True)
+
+
+class TriggerBotForm(forms.Form):
+    prompt_text = forms.CharField(
+        label="Prompt Text",
+        widget=forms.Textarea(attrs={"rows": 4}),
+        help_text="The prompt to send to the bot",
+        required=True,
+    )
+    experiment = forms.ModelChoiceField(
+        label="Select Chatbot",
+        queryset=Experiment.objects.none(),
+        required=True,
+        help_text="Select the chatbot to trigger",
+    )
+    start_new_session = forms.BooleanField(
+        label="Start a new session",
+        required=False,
+        initial=False,
+        help_text="End any previous sessions and start a new one",
+    )
+    session_data = forms.CharField(
```

Comment:
you could use `forms.JSONField` which will handle the serialization and validation for you:


```suggestion
    session_data = forms.JSONField(
        encoder=PrettyJSONEncoder,
```

(you'll need to add the `PrettyJSONEncoder` import)